newton/_src/sim/builder.py

```
@@ -7370,12 +7370,13 @@ def add_soft_mesh(
         rot: Quat,
         scale: float,
         vel: Vec3,
-        vertices: list[Vec3],
-        indices: list[int],
-        density: float,
-        k_mu: float,
-        k_lambda: float,
-        k_damp: float,
+        mesh: TetMesh | None = None,
```

Ruff (0.15.2)

[error] Undefined name `TetMesh`

(F821)